pkg/el/google-fluentd-catch-all-config.spec

```diff
@@ -0,0 +1,40 @@
+Summary: Catch-all configuration files for google-fluentd on GCE.
+Name: google-fluentd-catch-all-config
+Version: 0.1
+Release: 1%{?dist}
+License: Apache
```

Comment:
Which Apache license?  Also, you should use the short form from https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Good_Licenses